run.py

```diff
@@ -1,22 +1,56 @@
-##################
-# Note: this file should not be updated as it is a wrapper
+#!/usr/bin/env python3
 import json
-import time
-import pprint
 import sys
-import handler
 
-def get_stdin():
+import verifier
+
+ERROR= "VERIFIER_ERROR"
+ISSUES = "ISSUES"
+OK = "OK"
+
+
+def protocol(message, verify_function=verifier.verify):
+    try:
+        json.loads(message)
```

Comment:
handle the case where message == None. Currently, when this method i called via GET request with no body, i get 
`"BAD JSON: Expecting value: line 1 column 1 (char 0)"`
would be better to say
`"BAD JSON: null json"`
or something similar, indicating to the user, they sent null